build/images/deps/go-version

```diff
@@ -1 +1 @@
-1.17
+1.19
```

Comment:
Not related to this PR, but I am thinking we may use the same version from go.mod instead of this file in the future?